js/src/scatter.js

```diff
@@ -78,6 +95,13 @@ var ScatterView = widgets.WidgetView.extend( {
             visible: this.model.get("visible") && this.model.get("visible_markers")
             })
 
+        this.material_sprite = new THREE.RawShaderMaterial({
```

Comment:
New material -- basically add a #define USE_SPRITE

Reply:
please use sprite_material to be consistent with the rest of the naming (or change chose ;-) )